bin/transform-genbank

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env python3
+"""
+Parse the GenBank JSON load into a metadata tsv and a FASTA file.
+"""
+import argparse
+from pathlib import Path
+import fsspec
+import re
+import pandas as pd
+from typing import List
+
+
+# Note: 'sequence' should NEVER appear in this list!
+METADATA_COLUMNS = [  # Ordering of columns in the existing metadata.tsv in the ncov repo
+    'strain', 'virus', 'gisaid_epi_isl', 'genbank_accession', 'date', 'region',
+    'country', 'division', 'location', 'region_exposure', 'country_exposure',
+    'division_exposure', 'segment', 'length', 'host', 'age', 'sex',
+    'originating_lab', 'submitting_lab', 'authors', 'url', 'title',
+    'date_submitted'
+]
```

Comment:
I think we'd ideally import this list from a shared file for genbank and gisaid transformations.